tests/react/abortable.test.tsx

```diff
@@ -119,7 +119,7 @@ describe('abortable atom test', () => {
     expect(abortedCount).toBe(1)
   })
 
-  it('can abort on unmount', async () => {
+  it('does not abort on unmount', async () => {
```

Comment:
```suggestion
  it('does not abort on unmount (#2627)', async () => {
```

Reply:
This makes sense! Though I see you've already merged!

Reply:
Ah, I forgot to apply changes. 😓